CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+### Version 12.1.1
+
+2017-12-24
+
+[Compare changes](https://github.com/codevise/pageflow/compare/12-1-0...v12.1.1)
+
+##### Internals
+
+- Upgraded `paperclip` to version 4.3.7. This is currently the latest in the
+4.3 series and it contains deprecation warnings for the needed aws-sdk upgrade
+in paperclip 5. #923
+
 ### Version 12.1.0
```

Comment:
The changelog is a semi-generated file (items via [pr_log](https://github.com/tf/pr_log), sections/heading via [semmy](https://github.com/tf/semmy). Making changes in PRs makes this a bit harder and risks having merge conflicts.

Reply:
I did not know! I will revert these changes and trust that you will work your magic to generate everything?

Reply:
Maybe we can just include these snippets for the changelog in the PR description itself? Especially for things like manual update steps and breaking changes, well written instructions are nothing a tool can generate, of course.

Maybe we should have a wiki page briefly outlining our PR process? I also sometimes add [WIP] prefixes to PRs just to stay on top of what has to be reviewed etc. Without clearly having stated a process, I sometimes fear that changing the title like this might come across a bit passive aggressive. Like saying "Shame! This not done yet!" :smile: 

It would be nice if Github had some more first class support for this. The wip label would be a nice alternative, but, unless I'm mistaken, only users with write access to the repo can edit labels. That would mean (theoretical :wink:) outside contributors would not have a chance to remove the wip label once one of us set it.